variables.tf

```diff
@@ -55,3 +50,9 @@ variable "ec2amicreate_role_name" {
   description = "The name to assign the IAM role that allows allows this IAM user to create AMIs.  Note that a \"%s\" in this value will get replaced with the user_name variable."
   default     = "EC2AMICreate-%s"
 }
+
+variable "ssm_parameters" {
+  type        = list(string)
+  description = "The AWS SSM parameters that the IAM user needs to be able to read (e.g. [\"/example/parameter1\", \"/example/config\"])."
+  default     = []
+}
```

Comment:
Please add an "Optional parameters" comment block to separate them from the required parameters.

Reply:
That already exists here: https://github.com/cisagov/ami-build-iam-user-tf-module/blob/a392fdd82628bc6d5ce3d2d0ea56e7f8cc51771a/variables.tf#L12-L16

I moved this parameter to the bottom of the optional parameters because it is last alphabetically.

Reply:
I am a buffoon!